service/application/src/test/java/com/solace/maas/ep/event/management/agent/service/ScanServiceTests.java

```diff
@@ -339,4 +341,84 @@ public void testFindById() {
 
         assertThatNoException();
     }
+
+    @ParameterizedTest
+    @ValueSource(strings = {"COMPLETE", "FAILED", "TIMED_OUT"})
+    @SneakyThrows
+    public void testIsScanCompleteIsComplete(String scanStatus) {
```

Comment:
the test name is confusing, maybe `testScanFinalizedStatus` or testScanFinalStatus` is more appropriate here?